src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Tcp/ClientCredentialTypeTests.cs

```diff
@@ -174,4 +173,47 @@ public static void TcpClientCredentialType_Certificate_EchoString()
             ScenarioTestHelpers.CloseCommunicationObjects((ICommunicationObject)serviceProxy, factory);
         }
     }
+
+    [Fact]
+    [OuterLoop]
+    public static void TcpClientCredentialType_Certificate_CustomValidator_EchoString()
+    {
```

Comment:
That variation isn't in, frankly I wasn't even thinking about negative scenarios, should I add it here or do we need to put it in the 'ExpectedExceptions' area?